src/object/srv_obj.c

```diff
@@ -1713,29 +1715,80 @@ do_obj_ioc_begin(uint32_t rpc_map_ver, uuid_t pool_uuid,
 
 out:
 	dss_rpc_cntr_enter(DSS_RC_OBJ);
+	/** increment active request counter and start the chrono */
+	tls = obj_tls_get();
+	(void)d_tm_increment_gauge(&tls->ot_op_active[opc], 1, NULL);
+	ioc->ioc_start_time = daos_get_ntime();
```

Comment:
probably only record latency if rc == 0, since it only measure the latency for success IO.

Reply:
Yeah, but i thought that it is not worth it since this is on an error path anyway.

Reply:
oh, ok.